frontend/src/concepts/pipelines/content/createRun/RunForm.tsx

```diff
@@ -55,17 +55,16 @@ const RunForm: React.FC<RunFormProps> = ({ data, runType, onValueChange }) => {
   }, [latestVersion, onValueChange, updateInputParams]);
 
   return (
-    <Form
-      maxWidth="500px"
-      onSubmit={(e) => {
-        e.preventDefault();
-      }}
-    >
-      <FormSection id="run-section-project-name" title="Project">
-        <FormGroup label="Project">
-          <Text>{getProjectDisplayName(data.project)}</Text>
-        </FormGroup>
+    <Form onSubmit={(e) => e.preventDefault()} maxWidth="50%">
```

Comment:
Why did you set the max width to 50% here? I noticed when scaling the screen down that this makes the fields to truncated

Reply:
I'll revert since it looks like the page responsiveness isn't setup as I thought it was, but I think in general, when the jump-to-section is to the left of the form, taking up 50% of the screen makes more sense to me than an arbitrary pixel amount.

Reply:
ideally we have a width: 50% and a min width: 500px.... or just have breakpoints set up

but our ui is not really optimized for smaller displays anyway